ios/Podfile.lock

```diff
@@ -205,6 +205,8 @@ PODS:
     - React-cxxreact (= 0.61.5)
     - React-jsi (= 0.61.5)
   - React-jsinspector (0.61.5)
+  - react-native-background-timer (2.2.0):
```

Comment:
Just to make sure, did you mean to commit the `Podfile.lock` in this PR?

Reply:
Yes, it was modifying for me since the project setup on step "pod install" under ios folder, but I wanted to confirm before I commit the lock file changes.